doc/how_to/waveform_extractor_to_sorting_analyzer.rst

```diff
@@ -0,0 +1,572 @@
+    new_sorting_analyzer_path = "path_to/my_new_sorting_analyzer_folder"
+
+    extractor = load_waveform(folder = waveform_folder_path)
+    sorting_analyzer = extractor.sorting_analyzer
+    sorting_analyzer.save_as(folder = new_sorting_analyzer_path, format = "binary_folder")
+
+
+The above code creates a ``SortingAnalyzer`` folder at ``new_sorting_analyzer_path``.
+
+Dictionary between SortingAnalyzer and WaveformExtractor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section provides a dictionary for code, to help translate between a ``SortingAnalyzer``
+and a ``WaveformExtractor``, so that users can easily update old code. If you want to learn
+how to use ``SortingAnalyzer`` from scratch, the
+`Get Started <https://spikeinterface.readthedocs.io/en/latest/how_to/get_started.html>`_ guide
+and the `PostProcessing module documentation <https://spikeinterface.readthedocs.io/en/latest/modules/postprocessing.html>`_
+are better places to start.
+
+This section is split into four subsections:
+
+* `Create, load and save <#id2>`_
+* `Checking basic properties <#id3>`_
+* `Compute Extensions <#id4>`_
+* `Quality Metrics <#id5>`_
+
+Throughout this section, we assume that all functions have been imported into the namespace.
+E.g. we have run code such as ``from spikeinterface.postprocessing import extract_waveforms`` for each function. If you have imported
+the full package (ie you have run ``import spikeinterface.full as si``) you need to prepend all
+functions by ``si.``. If you have imported individual modules (ie you have run `import spikeinterface.postprocessing as spost`)
+we will need to prepend functions by the appropriate submodule name.
+
+
+In the following we start with a recording called `recording` and a sorting
+object called ``sorting``. We’ll then create, export, explore and compute things using a
+``SortingAnalyzer`` called ``analyzer`` and a ``WaveformExtractor`` called ``extractor``.
+We’ll do the same calculations for both objects. The WaveformExtractor code will be on
+the left while the SortingAnalyzer code will be displayed on the right:
+
+.. grid:: 2
+
+    .. grid-item::
+
+        WaveformExtractor
+        ^^^^^^^^^^^^^^^^^
+
+    .. grid-item::
+
+        SortingAnalyzer
+        ^^^^^^^^^^^^^^^
+
+Create, load and save
++++++++++++++++++++++
+
+First, create the object from a recording and a sorting.
+
+.. grid:: 2
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            extractor = extract_waveforms(
+                sorting = sorting,
+                recording = recording)
+
+    .. grid-item::
+
+        .. code-block:: python
+
+            analyzer = create_sorting_analyzer(
+                sorting = sorter,
+                recording = recording)
+
+
+By default, the object is stored in memory. Alternatively, we can save it locally at the point of creation by specifying a ``folder`` and a ``format``. Additionally, you can decide whether to use sparsity or not
```

Comment:
I would add a note here that in memory means that if you end the session before running `save_as` everything will be lost. Nothing worse then forgetting to save and losing a bunch of work :) You have the section below, but the goal according to Sam is in-memory is for testing conditions and iterating then saving. So actually explaining the intention of memory vs not is another change between the waveform extractor and the sorting analyzer